### ISSUE TITLE
Error when loading on Windows

### ISSUE THREAD

**Issue by joshuap** [honeybadger-io/honeybadger-python#36]
A customer emailed in to say that they are getting an error when loading honeybadger-python under Windows.

Code:

```python
from honeybadger import honeybadger
honeybadger.configure(api_key='62ff4aea')
raise Exception("This will get reported!")
```

Result:

```
Traceback (most recent call last):
File "C:\Users\Name\AppData\Local\Programs\Python\Python37-32\lib\site-packages\honeybadger\core.py", line 39, in exception_hook
self._send_notice(value, exc_traceback, context=self._get_context())
File "C:\Users\Name\AppData\Local\Programs\Python\Python37-32\lib\site-packages\honeybadger\core.py", line 22, in _send_notice
payload = create_payload(exception, exc_traceback, config=self.config, context=context)
File "C:\Users\Name\AppData\Local\Programs\Python\Python37-32\lib\site-packages\honeybadger\payload.py", line 102, in create_payload
'server': server_payload(config),
File "C:\Users\Name\AppData\Local\Programs\Python\Python37-32\lib\site-packages\honeybadger\payload.py", line 68, in server_payload
loadavg = os.getloadavg()
AttributeError: module 'os' has no attribute 'getloadavg'
Original exception was:
Traceback (most recent call last):
File "E:\Random Scripts\youtube-search-test.py", line 85, in <module>
raise Exception("This will get reported!")
Exception: This will get reported!
```

**Comment by Aulig** [comment]
Im also getting this issue on windows 10 with flask.```
 from honeybadger.contrib import FlaskHoneybadger

FlaskHoneybadger(app, report_exceptions=True)
```

**Comment by joshuap** [comment]
Thanks for the traceback, @Aulig. @demsullivan could you look into supporting Windows?

**Comment by joshuap** [comment]
For future reference, this was released in 0.3.0.
